tests/case_insensitivity.csv

```diff
@@ -11,3 +11,4 @@ base32hexpad, "td1imor3f41RMUSJCCG======"
 base32hexpadupper, "Td1imor3f41RMUSJCCG======"
 base36, "kfUvrsIvVnfRbjWaJo"
 base36upper, "KfUVrSIVVnFRbJWAJo"
+base45, "R+8d vd82ek4f.kea2"
```

Comment:
I don't think this is ok, base45 has a strict character set and it's upper-case

I agree with the discussion in https://github.com/multiformats/multibase/pull/123 that rfc9285 doesn't read like it leaves room for alternative case of its alphabet

```suggestion
```

Reply:
The case insensitivity test is to test if it is able to parse lowercase. We can throw an error on lowercase, or we can accept all cases, add a `toUpper` function on the input and decode it accordingly.

Honestly, if you look at `base36upper`, the name says `upper`, so technically `base36upper` is strictly upper. However, when a input is entered with an uppercase `K`, but lowercase letters after that, it will still accept and decode it. I think we have to handle `base45` in the same way. The only difference is that it hasn't `upper` in its name (yet).

Reply:
yeah, actually I'm going to go along with this. I don't think it's actually useful for it to be case insensitive, not like base32 which makes for nice domain names, but since there's not really a good reason that it _can't_ be, then I guess why not